src/views/Pathways/MentoShipTree/MentorShipTree.js

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect } from "react"
```

Comment:
it should be `MentorshipTree`. Check the filename. It is wrong.

also instead of importing MentorshipTree/MentorshipTree. It is better to have `MentorshipTree/index.js`